hack/testdata/multi-resource-json-modify.json

```diff
@@ -43,7 +43,7 @@
        "spec":{
          "containers":[{
            "name": "mock-container",
-           "image": "k8s.gcr.io/pause:3.5",
+           "image": "registry.k8s.io/pause:3.8",
```

Comment:
should we just be bumping to registry.k8s.io/pause:3.5?

Reply:
Per some discussion with @SergeyKanzhelev I was going for moving all to pause:3.8.  If I should cancel that, let me know and I'll drop those changes.

Reply:
per offline discussion: I'm +1 to either approach, I just want to be sure this is a conscious decision as opposed to because we're attempting to backport the previous PR. pause should be backwards compatible and this change is fine by me.

Reply:
I speak from ignorance, not knowing much about the pause container, or what changed between 3.5 and 3.8, but it seems like the lowest risk change is to just change the registry name, not the image content pointed to

Reply:
per more discussion I am removing the pause:3.8 standardization from this PR, to reduce risk and increase the likelihood of meeting a tight timeline for merging and releasing.